.gitignore

```diff
@@ -0,0 +1,4 @@
+node_modules
+.DS_Store
+build/*
```

Comment:
I've personally tried to start using `.` directories for things like this, but looking at it, `/dist` seems to be more standard.